src/viur/core/skeleton.py

```diff
@@ -319,11 +319,8 @@ def __ior__(self, other: dict | SkeletonInstance | db.Entity) -> SkeletonInstanc
                 new_entity[key] = value
             self.setEntity(new_entity)
         elif isinstance(other, SkeletonInstance):
-            new_entity = self.dbEntity or db.Entity()
-            # We're not overriding the key
-            for key, value in other.dbEntity.items():
-                new_entity[key] = value
-            self.setEntity(new_entity)
+            for key, value in other.accessedValues.items():
```

Comment:
But, why not all values and only accessed value?

Reply:
You mean the values form the `dbEntity` and the accessedValues ?